org.adempiere.base/src/org/idempiere/model/IMappedModelFactory.java

```diff
@@ -46,7 +46,8 @@
 	 */
 	void addMapping(String tableName, Supplier<Class<?>> classSupplier,
 			BiFunction<Integer, String, ? extends PO> recordIdFunction,
-			BiFunction<ResultSet, String, ? extends PO> resultSetFunction);
+			BiFunction<ResultSet, String, ? extends PO> resultSetFunction,
+			BiFunction<String, String, ? extends PO> recordUUFunction);
```

Comment:
the changes to IMappedModelFactory and the classes below is not needed as you have remove the uuid method from IModelFactory.

Reply:
@hengsin good catch, I missed it. Let me correct it.